common/src/java/com/zimbra/common/soap/MailConstants.java

```diff
@@ -1404,4 +1404,11 @@ private MailConstants() {
     public static final QName RECOVER_ACCOUNT_RESPONSE = QName.get(E_RECOVER_ACCOUNT_RESPONSE, NAMESPACE);
     public static final String A_RECOVERY_ACCOUNT = "recoveryAccount";
     public static final String A_CHANNEL = "channel";
+    
+    // HTML Constant
+    public static final String BREAK_LINE = "<br>";
+    public static final String SUCCESS = "Success";
```

Comment:
For Success, we already have key, why we are using new now?

Reply:
@deepakzimb I searched but didn't found out so created new.
can you help me with the one already defined, I'll use the same.

Reply:
After going through the code and checking with team.
E_ are for elements in xml
A_  attributes are attributes in xml
So we should keep this constant different without a prefix.